### DISCUSSION TITLE
Adding Max Star count to Media Summary

### DISCUSSION THREAD

**Discussion by leforet** [aaronwmorris/indi-allsky#1660] [Ideas]
Right now in the media > TimeLapse tab, it shows the Avg. Stars, Kindex, Aurora, Smoke.   I was wondering if we could add Max Stars to the list? I've noticed that sometimes the avg stars is off if a cloud bank rolls in.  The stars will go from a count of 1500+ to 20 and I will get a low average.

**Comment by aaronwmorris** [comment]
I will work on this.

**Comment by aaronwmorris** [comment]
Merged #1667 to add the maximum star calculations to the timelapse info.
